pyudev/monitor.py

```diff
@@ -34,7 +34,7 @@
 from threading import Thread
 from functools import partial
 
-from pyudev._util import ensure_byte_string
+from pyudev._util import ensure_byte_string, eintr_retry_call
```

Comment:
Please put all new imports on a separate line, alphabetically. This is just my aesthetic preference.